.github/actions/notify-build-status/action.yml

```diff
@@ -0,0 +1,115 @@
+name: notify-build-status
+description: "Notify the build status"
+
+inputs:
+  vaultUrl:
+    description: Vault URL
+    required: true
+  vaultRoleId:
+    description: Vault role ID
+    required: true
+  vaultSecretId:
+    description: Vault secret ID
+    required: true
+  slackChannel:
+    description: Slack channel
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - uses: hashicorp/vault-action@v2.4.2
+      with:
+        url: ${{ inputs.vaultUrl }}
+        roleId: ${{ inputs.vaultRoleId }}
+        secretId: ${{ inputs.vaultSecretId }}
+        method: approle
+        secrets: |
+          secret/observability-team/ci/stack-integartion-token integration_token | SLACK_BOT_TOKEN
```

Comment:
leftover?

```suggestion
```

Reply:
Oh yes, thank you for noticing: [001a698](https://github.com/elastic/apm-pipeline-library/pull/2048/commits/001a698eaa20f96f9bc3f5be53d2bb0156318cf7)